### PR TITLE
Code review rejects

### PR DESCRIPTION
**Pull request by dratler**
Hi @shs96c  here is Target PR rejects
related to  #6667
1. Fix typo's
2. use Selenium Session
3. Fix tests

#behalfOf_Intuit

### CODE REVIEW

File: java/client/src/org/openqa/selenium/devtools/target/model/AttachToTarget.java [hunk 1]
```diff
@@ -17,44 +17,42 @@
...
     this.sessionId = Objects.requireNonNull(sessionId, "sessionId is required");
-    this.targetId = Objects.requireNonNull(targetId, "targetId is required");
+    this.targetInfo = Objects.requireNonNull(targetInfo, "targetInfo is required");
     this.waitForDebugger = Objects.requireNonNull(waitForDebugger, "waitForDebugger is require");
   }
 
   private static AttachToTarget fromJson(JsonInput input) {
-    SessionId sessionId = null;
-    TargetId targetId = null;
+    SessionId sessionId = new SessionId(input.nextString());
```
Comment (shs96c) [inline java/client/src/org/openqa/selenium/devtools/target/model/AttachToTarget.java]: Shouldn't this be inside the loop? And I don't see a call to `input.beginObject`, so I'm a little surprised this works....

Comment (shs96c) [reply java/client/src/org/openqa/selenium/devtools/target/model/AttachToTarget.java]: Also, you added a `fromJson` method, so it should be fine to `input.read(SessionId.class)`

**Comment by dratler** [comment]
Hi Simon,
I have added the requested comments .
also I have open a new CR for GCD (there is one more CR on my behalf).
if you have other tasks I can assist please do I would like to .

Cheers
Shay

File: java/client/src/org/openqa/selenium/devtools/target/model/AttachToTarget.java (hunk 1 above)
Comment (dratler) [reply java/client/src/org/openqa/selenium/devtools/target/model/AttachToTarget.java]: this is how the Json Object is build .
I have tested it more the once

**Comment by shs96c** [comment]
I rebased this manually, merged and pushed as f323d601ae9f56e6bdfc3911523dadcc6bc29ab2. Thank you!

**Pull request closed without merging**